XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectServers.cs

```diff
@@ -222,6 +236,14 @@ private void EnabledRow(Host host, UpdateType type, int index)
                     return;
                 }
 
+                
+                if (poolOfOne != null && PoolForbidsAutomaticUpdates(poolOfOne))

```

Comment:
I think this is good here, as we need to do the previous checks first, such as license status or partially upgraded pools, thus XC will show the restart-related tooltip only if that is the only reason we can't do the auto (batch) update.

I am happy to rethink this, but I think a reason such as having a partially upgraded pool or an unlicesed pool or the pool is up-to-date shall be shown even if restarting was forbidden for any reason.